<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayer</class>
 <widget class="QWidget" name="MediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QVideoWidget" name="m_video" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_lblStatus"/>
     </item>
     <item>
      <widget class="PlainToolButton" name="m_btnPlayPause"/>
     </item>
     <item>
      <widget class="PlainToolButton" name="m_btnStop"/>
     </item>
     <item>
      <widget class="QSlider" name="m_slidProgress">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_lblTime"/>
     </item>
     <item>
      <widget class="PlainToolButton" name="m_btnVolume"/>
     </item>
     <item>
      <widget class="QSlider" name="m_slidVolume">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PlainToolButton" name="m_btnDownload"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlainToolButton</class>
   <extends>QToolButton</extends>
   <header>plaintoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
